.gitignore

```diff
@@ -15,3 +15,4 @@ fedmsg.d/hotness.py
 Vagrantfile
 *.retry
 .cache
+.vagrant/
```

Comment:
Since this doesn't have to do with docs, I recommend putting it in it's own pull request.

Reply:
Fixed